zyte_common_items/page_objects.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+
+import attrs
+from web_poet import ItemPage, ResponseUrl, Returns, field
+
+from .components import Metadata
+from .items import Product, ProductList
+
+
+@attrs.define
+class Page(ItemPage):
+    _url: ResponseUrl
+
+    @field
+    def metadata(self) -> Metadata:
+        """…"""
```

Comment:
```suggestion
        """Marks date and time when the page object created the item."""
```

Reply:
This is a leftover from testing https://github.com/scrapinghub/web-poet/issues/99, I need to remove it altogether, at the moment the docs need to be written on the reStructuredText files instead.